src/components/table/core/td.ts

```diff
@@ -363,6 +366,10 @@ export class TableData extends MutableElement {
   public override disconnectedCallback(): void {
     super.disconnectedCallback()
 
+    this.stopHoverCheck()
+    this.removeEventListener('pointerenter', this.onPointerEnter.bind(this))
+    this.removeEventListener('pointerleave', this.onPointerLeave.bind(this))
```

Comment:
this doesn't do what you think it does; `.bind(this)` returns a _new_ object, so you're removing a different function than you added, and therefore not actually removing it at all -- creating a memory leak

you want the `constructor()` to set `this.onPointerEnter = this.onPointerEnter.bind(this)`, replacing the unbound version, and then use `this.onPointerEnter` everywhere else in code